charts/perses/templates/tests/test-config.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.testFramework.enabled -}}
```

Comment:
I'm curious, how these tests work ? What is the requirements ? 
I see that it creates pod, sa, role, role binding. So it means that a pod is created at deployment time and removed ? 
If you deploy in production with let's say argocd, how Argocd will react on that? Will it recreate the tests on each sync ?

Reply:
The test resources (ServiceAccount, Role, RoleBinding, Deployment, ConfigMap) located in `templates/tests` are only created when you explicitly run `helm test <release-name>`. They won't be created during normal helm deployments or ArgoCD syncs. :)